### PR TITLE
Adsk Contrib - Add color space aliases

### PR DESCRIPTION
**Pull request by BernardLefebvre**
Aliases are added to color spaces as described in issue #1129 
Also adding a name to the config.

Signed-off-by: Bernard Lefebvre <bernard.lefebvre@autodesk.com>

### CODE REVIEW

**Review by zachlewis — COMMENTED** [review]
Nicely done, Autodesk. Multiple alases are a pleasant surprise, as is the config name attribute. Both seem insanely useful for reconciling bespoke conventions among configs / pipelines. I appreciate having a canonical method for retrieving canonical names, and similar pipeline-facilitating mechanisms built in to OCIO.

The test cases suggest that aliases work exactly how I'd like and expect. 


Thanks!

File: src/OpenColorIO/ColorSpaceSet.cpp [hunk 1]
```diff
@@ -109,21 +119,53 @@ class ColorSpaceSet::Impl
 
     void add(const ConstColorSpaceRcPtr & cs)
     {
-        const std::string csName = StringUtils::Lower(cs->getName());
-        if (csName.empty())
+        const char * csName = cs->getName();
```
Comment (michdolan) [inline src/OpenColorIO/ColorSpaceSet.cpp:122]: Should this still be lowercase, for the below comparison?

Comment (BernardLefebvre) [reply src/OpenColorIO/ColorSpaceSet.cpp:122]: StringUtils::Compare is not case sensitive

Comment (BernardLefebvre) [reply src/OpenColorIO/ColorSpaceSet.cpp:122]: StringUtils::Compare is not case sensitive

File: include/OpenColorIO/OpenColorIO.h [hunk 2]
```diff
@@ -291,6 +291,15 @@ class OCIOEXPORT Config
      */
     void validate() const;
 
+    /**
+     * Get/set a name string for the config.
+     *
+     * In combination with a color space name, this may be used to provide a more global reference
```
[the remark this answers is not in the document]
Comment (BernardLefebvre) [reply include/OpenColorIO/OpenColorIO.h:297]: discussed at TSC

File: src/OpenColorIO/ColorSpaceSet.cpp (hunk 1 above)
Comment (michdolan) [reply src/OpenColorIO/ColorSpaceSet.cpp:122]: Ah, ok. Thanks for the clarification.

**Review by hodoulp — APPROVED** [review]

**Pull request merged**